app/presenters/root_browse_page_presenter.rb

```diff
@@ -0,0 +1,30 @@
+class RootBrowsePagePresenter
+
+  def render_for_publishing_api
+    {
+      content_id: "8413047e-570a-448b-b8cb-d288a12807dd",
+      format: "mainstream_browse_page",
+      title: "Browse",
+      locale: 'en',
+      public_updated_at: Time.now.iso8601,
```

Comment:
I think `Time.now.utc` is a reasonable default in that case.  The content-store requires this field to be set, so we can't send `nil`.